### PR TITLE
Feature#46 implement google cloud storage

### PR DESCRIPTION
**Pull request by lucasvidela94**
# Description
What did you implemented/Why did you implemented this:
 - Implemented google cloud storage for avatar users.
 - Added google services to storage.rb. with our private keys.
 - Added google-cloud-storage to gem file.
 - Created migration for Active storage.

### CODE REVIEW

**Review by AgustinPalmaM — DISMISSED** [review]
LGTM

**Review by AgustinPalmaM — DISMISSED** [review]
LGTM

File: config/cloudstorage.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    }
+  
+    # The ID to give your GCS bucket
+    bucket_name = "sapec-images"
+  
+    # Creates the new bucket
+    bucket = storage.create_bucket bucket_name
+  
+    puts "Bucket #{bucket.name} was created."
+  end
```
Comment (YilKanda) [inline config/cloudstorage.rb:17]: Add a blank line

File: config/cloudstorage.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+def quickstart bucket_name:
+    # Imports the Google Cloud client library
+    require "google/cloud/storage"
+  
+    # Instantiates a client
+    storage = Google::Cloud::Storage.new{
+
```
Comment (YilKanda) [inline config/cloudstorage.rb:7]: Remove this blank space

File: config/cloudstorage.rb (hunk 1 above)
Comment (lucasvidela94) [reply config/cloudstorage.rb:17]: Thanks Yil i will add it right now.

**Review by EdwardMD — APPROVED** [review]
LGTM!

**Review by acorzo — APPROVED** [review]
LGTM 👍

**Pull request merged**